src/components/highlighter/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import Prism from 'prismjs';
+import Highlight from 'prism-react-renderer';
+import { Box, Flex, useTheme } from '@blockstack/ui';
+
+import { Token, GetTokenProps, RenderProps } from '@components/highlighter/types';
+import { theme } from '@components/highlighter/prism-theme';
+
+const lineNumberWidth = 60;
+const getLineNumber = (n: number) => (n + 1 < 10 ? '0' : '') + (n + 1);
+
+const Tokens = ({
```

Comment:
Can we give `Token` a more explicit name, something like `GrammaticalToken`?  Normally would be fine, but given the context of the app, with NonFungibleTokens and things like this, can get confusing.

Reply:
sure, much of this was taken from the way prism and the prism-react-renderer name thing, but your points make sense.